testing.postfixTemplates

```diff
@@ -20,3 +20,18 @@
 
 .assertLen : assert length
     NIL      →  assert.Len(t, $expr$, $VAR0$, "$end$")
+
+
+## create tase case ##
+
+.test : generate test case
```

Comment:
there is already a live template for `test` (eg If you type `test` somewhere in the editor). Again, is there an advantage to include it here as well?